airflow/operators/trigger_dagrun.py

```diff
@@ -89,6 +89,7 @@ class TriggerDagRunOperator(BaseOperator):
         (default: 60)
     :param allowed_states: List of allowed states, default is ``['success']``.
     :param failed_states: List of failed or dis-allowed states, default is ``None``.
+    :param soft_fail: Set to true to mark the task as SKIPPED on DagRunAlreadyExists
```

Comment:
`soft_fail` is not a suitable name because it does not cover all types of failures, and will confuse users.

Reply:
you are right , what about `skip_on_already_exist` ?

Reply:
Looks better, or `skip_when_already_exists`